lib/elixir/unicode/unicode.ex

```diff
@@ -5,313 +5,422 @@
 # 1. Replace UnicodeData.txt by copying original
 # 2. Replace PropList.txt by copying original
 # 3. Replace SpecialCasing.txt by copying original and removing conditional mappings
-# 4. Replace GraphemeBreakTest.txt by copying original auxiliary/GraphemeBreakTest.txt
-# 5. Replace GraphemeBreakProperty.txt by copying original auxiliary/GraphemeBreakProperty.txt
-# 6. Append Extended_Pictographic from emoji/emoji-data.txt to the end of GraphemeBreakProperty.txt
-# 7. Update String.Unicode.version/0 and on String module docs (version and link)
-# 8. make unicode
-# 9. elixir lib/elixir/unicode/graphemes_test.exs
+# 4. Update String.Unicode.version/0 and on String module docs (version and link)
+# 5. make unicode
+
+data_path = Path.join(__DIR__, "UnicodeData.txt")
```

Comment:
I have moved properties.ex to this file and renamed String.Unicode to String.Casing. Other than that, the contents are strictly the same.

Reply:
The other way around, right?

Reply:
Correct, renamed Casing to Unicode.